tasks/c/easy/sum.c

```diff
@@ -2,10 +2,28 @@
 
 #include <stdio.h>
 
-// TODO: Implement the sum function
+int sum(int a, int b){
+    int total  = a + b;
+    return total;
+}
```

Comment:
Looks good! There is no need to add any extra variables so you can just return a + b in that function but that works too.